src/node_webstorage.cc

```diff
@@ -572,7 +570,7 @@ static Intercepted StorageSetter(Local<Name> property,
   Storage* storage;
   ASSIGN_OR_RETURN_UNWRAP(&storage, info.This(), Intercepted::kNo);
 
-  if (storage->Store(property, value)) {
+  if (storage->Store(property, value).IsJust()) {
     info.GetReturnValue().Set(value);
   }
```

Comment:
`SetFalse()` doesn't exist in V8 12.4, so I fixed the conflict to keep the behavior identical.